tortoise/verifying_tortoise.go

```diff
@@ -498,7 +506,7 @@ func (t *turtle) calculateExceptions(
 		// support a block and we disagree, we need to add a vote against here.
 		// TODO: this is not currently possible since base block opinions aren't indexed by layer. See
 		//   https://github.com/spacemeshos/go-spacemesh/issues/2424
-		//for b, v := range baseBlockOpinion.BlockOpinions {
+		//for b, v := range baseBlockOpinion[layerID] {
```

Comment:
are you planning to add against diff in a follow-up PR?

Reply:
Either in a follow-up one or in this one if I fix the broken by this addition tests until merging and if the amount of changes is tiny